x/ibc/07-tendermint/types/evidence.go

```diff
@@ -63,7 +63,8 @@ func (ev Evidence) String() string {
 
 // Hash implements Evidence interface
 func (ev Evidence) Hash() tmbytes.HexBytes {
-	bz := SubModuleCdc.MustMarshalBinaryBare(ev)
+	// TODO use submodule cdc
```

Comment:
using `SubModuleCdc` here was throwing a proto codec error. Given that the direct followup pr will fix this I thought this is fine for now

Reply:
Why is this erroring? Because Evidence hasn't transitioned to proto yet?

Reply:
That sounds right, I didn't want to spend the time investigating the issue since it wasn't pressing for this pr. I probably need to add the `RegisterInterfaces` call from `x/evidence`